app/waf/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build with:
+#   docker build . -t f5devcentral/f5-wafaas-openstack:latest
+
+# upload image with (authorized user only)
+#   docker push f5devcentral/f5-wafaas-openstack:latest
+
+FROM node:11.6.0
```

Comment:
What is the size of this image? Did you compare it with any other image like node:11-alpine?

Reply:
11.6.0 is big. That should be good to use 11-alpine after testing.